realsense2_camera/src/parameters.cpp

```diff
@@ -82,6 +82,10 @@ void BaseRealSenseNode::getParameters()
     _base_frame_id = _parameters->setParam<std::string>(param_name, DEFAULT_BASE_FRAME_ID);
     _base_frame_id = (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_" << _base_frame_id)).str();
     _parameters_names.push_back(param_name);
+
+    param_name = std::string("data_processing_mode");
```

Comment:
maybe `accelerate_with_gpu`  , on/off``?
Or as is with values `CPU/GPU` ?

Reply:
Changed the param name to `accelerate_with_gpu` and declared this param as `int` not `bool`. Because, if in case in future, we need to add new values (like for CUDA, etc.,), it can be added easily to the `int` param.
@Nir-Az, your thoughts?

Reply:
Not sure CUDA is replacing GPU, let's make sure how it works in LibRS